lib/index.js

```diff
@@ -32,7 +34,13 @@ function main({ port, client } = {}) {
   let filters = {};
   if (config.accept) {
     debug('loading rules from %s', config.accept);
-    filters = require(path.resolve(process.cwd(), config.accept));
+    const acceptLocation = path.resolve(process.cwd(), config.accept);
+
+    if (acceptLocation.endsWith(".yaml") || acceptLocation.endsWith(".yml")) {
+        filters = yaml.safeLoad(fs.readFileSync(acceptLocation, 'utf8'))
+    } else {
+        filters = require(acceptLocation);
+    }
```

Comment:
Those are valid points. 👍 

Our thinking here is that being more strict about what we accept is a good thing. It will reduce the chances of things that could go wrong in the loading of the config.

Removing this logic statement here would be another win as it would simplify the code, making it easier to understand what is accepted (anything that is valid YAML, including valid JSON).